prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudication-container/adjudication-container.component.ts

```diff
@@ -331,6 +329,7 @@ export class AdjudicationContainerComponent implements OnInit {
       )
       .subscribe((enableEnrollee: HttpEnrollee) => {
```

Comment:
So all the places but two (toggle always manual / assign TOA) we call these two methods which both have get enrolleeById
![image](https://user-images.githubusercontent.com/59669288/146274252-5dd2c490-0f4b-4f7d-b506-fc0e44a261eb.png)
Probably easiest thing to so is not call it in the adjudicationActionPipe and just use the update enrollee to handle updating the enrollee

Reply:
Done